app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEdit.scala.html

```diff
@@ -219,9 +235,29 @@
     },
     attributes = Map("rows" -> "5", "autofocus" -> "true")))
 
+
+  <label class="govuk-label">@messages("edit-set.record.related-material")</h3>
```

Comment:
Rather than a label this should be a h3 as per the prototype, e.g. `<h3 class="govuk-heading-s">Related material</h3>`

Reply:
Done - I used the label class before because the other labels are not bold but I imagine these will be changed to match the prototype

Reply:
I think that we're using the `Label()` function in most cases.